merkle-tree/src/insurance_engine.rs

```diff
@@ -0,0 +1,24 @@
+use {
```

Comment:
I like the change overall, I am just curious why is this wile called insurance_engine.rs? Just a copy-paste relict?

Reply:
I was not sure about name here. First I was thinking to place it under `lib.rs` directly but I my thoughts then went in direction that `lib.rs` to be a generic merkle tree stuff and then there could be added some modules that work with some specific implementation. Here the tree node comes from insurance engine module so I took that name. It was intentional but probably not good.

Would you have idea how to do better? Removing the file? Change to a better name like `validator_bonds` maybe?

Reply:
Now I see. Yes, probably validators_bonds_claim or similar would make sense.

Reply:
:ok_hand: I used the name `psr_claim.rs`
(PR code forced to check with `main` branch where update is related to contract deploy)